### DISCUSSION TITLE
[Solved] Only works with the GitHub client rather than any Git client.

### DISCUSSION THREAD

**Discussion by Akronymus-git** [ThioJoe/YT-Spammer-Purge#324] [General]
As this repo uses .github rather than .git it is incompatible with a "normal" git client.

**Comment by Kurdakov** [comment]
what?
I'm sorry but it works with normal git clients either on windows or linux.
So what is your specific problem or did you write without even trying?

**Comment by Akronymus-git** [comment]
When using the git cli it gives me the error that this is not a valid repo, as it expects a .git folder. Where this project uses a .github folder. Using the git gui rather than git cli to make it more visual.
![image](https://user-images.githubusercontent.com/15932399/147970693-94bd7d61-c519-43a0-a761-42991e6798f6.png)

**Reply by Kurdakov** [reply]
when you clone with git it creates .git directory.
If you just downloaded archive - a .github directory contains files related to github interface. But no git will work either because it is not cloned repository

**Reply by Akronymus-git** [reply]
Huh, it seems I must have mistyped something before, then. As originally I couldn't get it to clone the repo with the error that it isn't a valid one. Probably was copying the wrong URL to clone from. Thanks for clearing it up for me.
